.eslintrc.json

```diff
@@ -47,15 +43,10 @@
     "react/jsx-filename-extension": [2, { "extensions": [".js", ".jsx", ".ts", ".tsx"] }],
     "@typescript-eslint/ban-ts-comment": ["error", { "ts-ignore": "allow-with-description" }],
     "no-shadow": "off",
-    "@typescript-eslint/no-shadow": ["error"]
+    "@typescript-eslint/no-shadow": ["error"],
+    "@typescript-eslint/no-unused-expressions": ["warn", { "allowShortCircuit": true, "allowTernary": true }]
```

Comment:
But you still have changed the code. We should either not have the exception to this rule or change the code, not both

Reply:
I see. I gonna revert the change from a &&b to if(a) b